tests/exception-policy-applayer-01/suricata.yaml

```diff
@@ -1,6 +1,10 @@
 %YAML 1.1
 ---
 
+stats:
+  enabled: yes
+  interval: 8
```

Comment:
Wondering if this should be a shorter interval, so we could possibly see stats records with and without the exception policy stats incremented.

Reply:
I think there are some implementation issues with reducing it, so lets not worry about it now